media_driver/linux/common/os/i915/include/intel_chipset.h

```diff
@@ -220,15 +220,21 @@
 
 #define PCI_CHIP_COFFEELAKE_S_GT1_1     0x3E90
 #define PCI_CHIP_COFFEELAKE_S_GT1_2     0x3E93
+#define PCI_CHIP_COFFEELAKE_S_GT1_3     0x3E99
 #define PCI_CHIP_COFFEELAKE_S_GT2_1     0x3E91
 #define PCI_CHIP_COFFEELAKE_S_GT2_2     0x3E92
 #define PCI_CHIP_COFFEELAKE_S_GT2_3     0x3E96
 #define PCI_CHIP_COFFEELAKE_H_GT2_1     0x3E9B
+#define PCI_CHIP_COFFEELAKE_H_GT1_1     0x3E9C
```

Comment:
This device ID is not yet known to the kernel. Wait till https://patchwork.freedesktop.org/patch/282347/ will land. Please, provide a separate PR.

Reply:
Since this patch is submitted, keep this device ID.

Reply:
Added the device ID needed by #507